internal/lock.go

```diff
@@ -89,7 +176,7 @@ func strToFileMode(perm string) (os.FileMode, error) {
 
 // Download gets all the resources in this lock file and moves them to
 // the destination directory.
-func (l *Lock) Download(dir string, tags []string, notags []string, perm string, status bool) error {
+func (l *Lock) Download(dir string, tags []string, notags []string, perm string, someBool bool) error {
```

Comment:
What's this change, why the name change?

Reply:
I didn’t really change it. When I was testing, the function needed five arguments, so I added a temporary one called someBool. Later, when I added Dylan’s code, I saw it should be status, so I fixed it.